### PR TITLE
109 create cohorts page for teachers

### PR DESCRIPTION
**Pull request by Haroogle**
Basic outline for chorts page if user is teacher

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -70,6 +77,7 @@ export default App
 
 const AuthenticateUser = ({ children, redirectPath = '/' }) => {
   if (!isLoggedIn()) {
+    console.log('user is not logged in')
```
Comment (dogezen) [inline src/App.js:80]: please remove this console log

File: src/components/Cohorts/CohortsPage/CohortsPageTeacher.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import { useEffect, useState } from 'react'
+import client from '../../../utils/client'
+import Header from '../../Header/Header'
+import CohortsList from '../../cohortsList/CohortsList'
+import './style.css'
+
+function CohortsPageTeachers({ loggedInUser }) {
+  console.log(loggedInUser)
```
Comment (dogezen) [inline src/components/Cohorts/CohortsPage/CohortsPageTeacher.js:8]: please remove this

File: src/components/Cohorts/CohortsPage/CohortsPageTeacher.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  }, [])
+
+  useEffect(() => {
+    client
+      .get('/cohorts')
+      .then((res) => setCohorts(res.data))
+      .catch((err) => console.log(err.response))
+  }, [])
+  console.log(users)
+  console.log(cohorts)
```
Comment (dogezen) [inline src/components/Cohorts/CohortsPage/CohortsPageTeacher.js:25]: please remove these console logs

File: src/components/Cohorts/CohortsPage/CohortsPageTeacher.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  return (
+    <body>
+      <header>
```
Comment (dogezen) [inline src/components/Cohorts/CohortsPage/CohortsPageTeacher.js:28]: not sure you need to wrap <Header> component in <header>, I would remove <header>

File: src/components/Cohorts/CohortsPage/CohortsPageTeacher.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  useEffect(() => {
+    client
+      .get('/cohorts')
+      .then((res) => setCohorts(res.data))
+      .catch((err) => console.log(err.response))
+  }, [])
+  console.log(users)
+  console.log(cohorts)
+  return (
+    <body>
```
Comment (dogezen) [inline src/components/Cohorts/CohortsPage/CohortsPageTeacher.js:27]: change <body> to <>; we already have a <body> tag (see public/index.html)

**Review by dogezen — DISMISSED** [review]

**Comment by Logi13** [comment]
So you will need to fix the issue:
`temporaryAddPopup` is assigned a value but never used. You need to decide who will fix that. If that variable is not used then you need to remove it or if it is meant to be used use it

**Review by Logi13 — APPROVED** [review]

**Review by lucianosimoni — APPROVED** [review]

**Pull request merged**